build/configs/rtl8721csm/rtl8721csm_make_bin.sh

```diff
@@ -127,14 +127,20 @@ function copy_bootloader()
 
 	source ${CONFIG}
 
+	BL_VER=3
+
+	if [ "${CONFIG_USE_BP}" == "y" ];then
+		BL_VER=4
+	fi
+
 	if [ "${CONFIG_AMEBAD_TRUSTZONE}" != "y" ];then
 		echo "========== Copy_bootloader for TZ disabled =========="
-		cp $BOOT_PATH/km4_boot_all.bin $BINDIR/km4_boot_all.bin
+		cp $BOOT_PATH$BL_VER/km4_boot_all.bin $BINDIR/km4_boot_all.bin
```

Comment:
`bootloader3` and `bootloader4` are difficult to know what they are and what difference is.
How about `bootloader_fixed` and `bootloader_bp` or `bootloader_dynamic`?

Reply:
Bootloader3 uses fixed kernel regions and bootloader4 uses bp to get kernel regions dynamically. I want to represent in the names. How do you think?

Reply:
Done, rename them to ```bootloader_dynamic_partitions``` and ```bootloader_fixed_partitions```